src/Certify.Shared/Management/PluginManager.cs

```diff
@@ -128,38 +139,89 @@ public void LoadPlugins(List<string> includeSet)
 
             if (includeSet.Contains("Licensing"))
             {
-                LicensingManager = LoadPlugin<ILicensingManager>("Plugin.Licensing.dll", typeof(ILicensingManager)) as ILicensingManager;
+                LicensingManager = LoadPlugin<ILicensingManager>("Plugin.Licensing.dll");
 
             }
 
             if (includeSet.Contains("DashboardClient"))
             {
-                DashboardClient = LoadPlugin<IDashboardClient>("Plugin.DashboardClient.dll", typeof(IDashboardClient)) as IDashboardClient;
+                DashboardClient = LoadPlugin<IDashboardClient>("Plugin.DashboardClient.dll");
             }
 
             if (includeSet.Contains("DeploymentTasks"))
             {
-                // TODO: wildcard filename match
-                var deploymentTaskPlugin = LoadPlugin<IDeploymentTaskProviderPlugin>("Plugin.DeploymentTasks.Core.dll", typeof(IDeploymentTaskProviderPlugin)) as IDeploymentTaskProviderPlugin;
-                DeploymentTaskProviders = new List<IDeploymentTaskProviderPlugin>
-                {
-                    deploymentTaskPlugin
-                };
-
-                var azure = LoadPlugin<IDeploymentTaskProviderPlugin>("Plugin.DeploymentTasks.Azure.dll", typeof(IDeploymentTaskProviderPlugin)) as IDeploymentTaskProviderPlugin;
-                DeploymentTaskProviders.Add(azure);
-
+                var deploymentTaskProviders = new List<IDeploymentTaskProviderPlugin>();
+                DeploymentTaskProviders = deploymentTaskProviders;
+                var core = LoadPlugin<IDeploymentTaskProviderPlugin>("Plugin.DeploymentTasks.Core.dll");
```

Comment:
It isn't; I did it this way to preserve load order (so that the Core and Azure tasks are always found before any provided by third parties). If you're confident there won't be any ID conflicts, loading solely by wildcard is fine.